src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -42,6 +157,8 @@ def bbox_filter_gtfs(
 
     Optionally filter to a list of given dates.
 
+    WARNING: THIS FUNCTION IS DEPRECATED AND WILL INEVITABLY BE REMOVED.
```

Comment:
Not sure we need to deprecate before an official release. Perhaps other team members have requested this. In which case, we should raise a deprecation warning.

Reply:
Warning added